### PR TITLE
Added package for Slackware

### PR DESCRIPTION
**Pull request by dslackw**
Signed-off-by: Dimitris Zlatanidis <d.zlatanidis@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ Have fun with it! PRs are welcome. Check out [#1](https://github.com/jarun/nnn/i
 - [Debian](https://packages.debian.org/search?keywords=nnn&searchon=names&exact=1)
 - [Homebrew](http://formulae.brew.sh/formula/nnn)
 - [NixOS](https://github.com/NixOS/nixpkgs/tree/master/pkgs/applications/misc/nnn) (`sudo nix-env -i nnn`)
+- [Slackware] (http://slackbuilds.org/repository/14.2/system/nnn/)
```
Comment (jarun) [inline README.md]: The extra space in between breaks Markdown. I have added the link myself.

**Comment by jarun** [comment]
Added it myself.

**Comment by jarun** [comment]
Would you be kind enough to submit [imgp](https://github.com/jarun/imgp) to Slackware?

**Comment by dslackw** [comment]
Done! [imgb SBo](http://slackbuilds.org/repository/14.2/graphics/imgp/)

**Comment by jarun** [comment]
Thank you so much!

**Pull request closed without merging**